src/main/java/com/openmrs/migrator/MigratorApplication.java

```diff
@@ -104,4 +109,13 @@ private void executeSetupCommand() throws IOException {
     bootstrapService.createDirectoryStructure(dirList);
     bootstrapService.populateDefaultResources(pdiFiles);
   }
+
+  private void setMigratorHome() {
+    String home = System.getProperty(MIGRATOR_HOME);
+    if (home == null) {
```

Comment:
currently we have both settings.properties and pdiresources/ copied into the working directory(PWD) when running the migrator command. If this change is intending to allow the user to manually set where we should look then i think it should rather setup and lookup things from the newly set $MIGRATOR_HOME.

My suggestion for possible values of $MIGRATOR_HOME would be;
- use $MIGRATOR_HOME if set
- use either $PWD or $USER_HOME (the team chooses) if $MIGRATOR_HOME is not set
That way we only know things are either in $MIGRATOR_HOME or (either $PWD or $USER_HOME), that means the setup command should populate settings.properties and pdiresources/ into $MIGRATOR_HOME and then all other retrievals for those would point to $MIGRATOR_HOME when run command is invoked.
But what if run is invoked before setup, we should require setup first, but for unit tests, we can allow them to look else where through the lib resources so that we can easily stabilise on their behaviour. thus one last possible option for $MIGRATOR_HOME would be java's getResource.